### PR TITLE
Master

### PR DESCRIPTION
**Pull request by chrislabeard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Bloom Starter
 
-This is a starter kit for React/Redux projects.
+This is a starter kit for react/redux projects.
```
Comment (inkblotty) [inline README.md]: These changes need to persist in the README

File: src/js/presentation/forms/new-user-form.jsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ class NewUserForm extends React.Component {
               value={ formData.email ? formData.email.value : '' } onChange={ props.updateForm } validateAs='email' />
             <TextInput placeholder='Enter 2FA Pin Code (required)' id='twoFactorSecret' name='twoFactorSecret' label='Two Factor Pin Code'
               value={ formData['twoFactorSecret'] ? formData['twoFactorSecret'].value : '' } onChange={ props.updateForm } />
-            <Button onClick={ this.toggleStage } text='Continue' className='Btn--alt LoginForm-submit-button u-justify-center' disabled={ !allowContinue } />
+            <Button onClick={ this.toggleStage } text='Continue' className='Btn--alt LoginForm__submit-button u-justify-center' disabled={ !allowContinue } />
```
Comment (inkblotty) [inline src/js/presentation/forms/new-user-form.jsx]: This fix needs to be kept too.

File: src/styles/core/_mixins.scss [hunk 3]
```diff
@@ -31,3 +31,77 @@ aka functions to output css
...
   transition: $args;
 }
+
+@mixin button-variant($color, $background, $border) {
+  color: $color;
+  background-color: $background;
+  border-color: $border;
+
+  &:focus,
+  &.focus {
```
Comment (inkblotty) [inline src/styles/core/_mixins.scss]: These should be .is-focus, .is-active, etc. for the class versions to match SUIT naming.

File: src/styles/main.scss [hunk 4]
```diff
@@ -21,7 +21,7 @@ main {
   min-height: calc(100vh - #{$header-height} - #{$footer-height});
 }
 
-.App-container {
+.app-container {
```
Comment (inkblotty) [inline src/styles/main.scss]: This guy needs to stay as .App-container.

File: src/styles/core/_mixins.scss [hunk 5]
```diff
@@ -31,3 +31,77 @@ aka functions to output css
...
+    &:focus,
+    &.focus {
+      color: $color;
+      background-color: darken($background, 17%);
+          border-color: darken($border, 25%);
+    }
+  }
+  &:active,
+  &.active,
+  .open > &.dropdown-toggle {
```
Comment (inkblotty) [inline src/styles/core/_mixins.scss]: `is-open` for SUIT

File: src/styles/core/_mixins.scss [hunk 6]
```diff
@@ -31,3 +31,77 @@ aka functions to output css
...
+
+@mixin alert-variant($background, $border, $text-color) {
+  background-color: $background;
+  border-color: $border;
+  color: $text-color;
+
+  hr {
+    border-top-color: darken($border, 5%);
+  }
+  .alert-link {
```
Comment (inkblotty) [inline src/styles/core/_mixins.scss]: .Alert-link

**Review by inkblotty — CHANGES_REQUESTED** [review]
Just a few naming convention things and fixes that were reverted. Looks good 👍

File: src/styles/main.scss (hunk 4 above)
Comment (chrislabeard) [reply src/styles/main.scss]: Hmm, I don't remember touching that but I'll change it back.

File: src/js/presentation/forms/new-user-form.jsx (hunk 2 above)
Comment (chrislabeard) [reply src/js/presentation/forms/new-user-form.jsx]: Another file I don't remember touching.

**Review by inkblotty — APPROVED** [review]

**Pull request merged**